pkg/provisioningv2/rke2/planner/planner.go

```diff
@@ -222,6 +222,15 @@ func (p *Planner) Process(cp *rkev1.RKEControlPlane, status rkev1.RKEControlPlan
 		return status, ErrWaitingf("%s/%s: releaseData nil for version %s", cp.Namespace, cp.Name, cp.Spec.KubernetesVersion)
 	}
 
+	// we can't enforce this condition during initial provisioning, all nodes need to come up
```

Comment:
Can you relocate this after the pause check for the CAPI cluster i.e. after the 
```
	if !capiCluster.DeletionTimestamp.IsZero() {
//...
	}
```
block?

Reply:
Okay, its been moved. I wanted to check though, is checking the `Reconciled` condition enough to ensure that we've undergone initial provisioning? I would think so, but I wouldn't be surprised if I've missing something regarding how this status is set during initial provisioning.